### ISSUE TITLE
problematic pitchmark's call in make_f0_pm need in voicing

### ISSUE THREAD

**Issue by ddavout** [festvox/festvox#38]
https://github.com/festvox/festvox/blob/3be8dd4e3d2ac680d8d6b33fe6212322636b45c8/src/intonation/make_f0_pm#L98
but 'pitchmark' has a file name as input

```
    F0MIN=50
    F0MAX=200
    F0MEAN=110
    # >....<
    F0_ARGS=`echo $F0MIN $F0MAX $F0MEAN | awk '{printf("-min %f -max %f -def %f",1.0/$2,1.0/$1,1.0/$3)}'`
    PM_ARGS='-wave_end -lx_lf 140 -lx_lo 111 -lx_hf 80 -lx_ho 51 -med_o 0'
    $ESTDIR/bin/ch_wave -scaleN 0.9 wav/$fname.wav -F 16000 | 
    $ESTDIR/bin/pitchmark -o pm_unfilled/$fname.pm -otype est $PM_ARGS $F0_ARGS
```
will raise

>     Write access failed for temporary file
>     Cannot open file TMP/est_868612_00000 as tokenstream
>     Wave load: can't open file "TMP/est_868612_00000"
>     Cannot recognize file format or cannot access file: "-"

    
    Could it be something like ?
```
    # pitchmark has a file name as input
    "$ESTDIR"/bin/ch_wave -scaleN 0.9 -F 16000 wav/"$fname".wav -o tmp_"$fname"
    "$ESTDIR"/bin/pitchmark -o pm_unfilled/"$fname".pm -otype est  $PM_ARGS $F0_ARGS tmp_"$fname"
```

**Comment by Pallas1303** [comment]
Output of:

```bash
$ESTDIR/bin/ch_wave -scaleN 0.9 wav/$fname.wav -F 16000 | 
```

Is a input of pitchmark.

**Comment by ddavout** [comment]
yes and that's part of the probleme as pitchmark has "normally" a filename as input

Envoyé avec la messagerie sécurisée [Proton Mail.](https://proton.me/)

**Comment by Pallas1303** [comment]
Input of pitchmark it can be the output (stdout?) of a command. 

**|** makes this.

**Comment by ddavout** [comment]
/bin/ch_wave ~/tempo/sample.wav  | head -n 1
RIFFʇWAVEfmt �>}data��������$P�N������a0����!P�_���(��7���������

**Comment by Pallas1303** [comment]
I thinking that "-" is for how use other commads. How -itype, -o and others. 

See more: [Speech Tools manual in cstr site](https://www.cstr.ed.ac.uk/projects/speech_tools/manual-1.2.0/x444.htm)

Stranger the site of festvox.org not load for my browser. The site was it bought?

**Comment by saikrishnarallabandi** [comment]
site is up when i checked today.
